cpp/doxygen/developer_guide/BENCHMARKING.md

```diff
@@ -54,3 +54,26 @@ reaches its saturation bottleneck, whether that bottleneck is bandwidth or compu
 sets larger than this point is generally not helpful, except in specific cases where doing so
 exercises different code and can therefore uncover regressions that smaller benchmarks will not
 (this should be rare).
+
+## Running and Comparing NVBench Benchmarks in libcudf
+
+### Running Benchmarks
+
+NVBench executables are built under the `cpp/build/latest/benchmarks` directory. Each benchmark is compiled
```

Comment:
Do we need to mention that benchmarks are not built by default unless enabled via `BUILD_BENCHMARKS=ON` CMake arg?

Reply:
Done in https://github.com/rapidsai/cudf/pull/18548/commits/be01b10d075cca712c9b69ca5f79fe8e851864e5